easybuild/easyconfigs/e/ESMValTool/ESMValTool-1.1.0-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,194 @@
+easyblock = 'Binary'
+
+name = 'ESMValTool'
+version = '1.1.0'
+versionsuffix = '-Python-%(pyver)s'
+hdf5ver = '1.8.19'
+
+homepage = 'https://www.esmvaltool.org/'
+description = """The Earth System Model eValuation Tool (ESMValTool)
+ is a community diagnostics and performance metrics tool
+ for the evaluation of Earth System Models (ESMs) that
+ allows for routine comparison of single or multiple models,
+ either against predecessor versions or against observations."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ESMValGroup/ESMValTool/archive/']
+sources = [{'filename': SOURCE_TAR_GZ, 'download_filename': 'v%(version)s.tar.gz'}]
+checksums = ['457d5a7e72165a31532b335c42b925abaaa48e959d32b123463df77318c73c36']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('NCL', '6.4.0'),
+    ('R', '3.4.3', '-X11-20171023-HDF5-%s' % hdf5ver),
```

Comment:
@hajgato Is it really necessary to add another easyconfig for `R`?

Reply:
Yes. `netCDF` with `HDF5 1.8` needed for couple geo stuff, so most of them done with this combo.

Reply:
@boegel ^

Reply:
I am telling for years: we should be able to share the extensions between different R/Python/YouNameIt easyconfigs. for example 1 R-3.4.3.extension could be used by all R-3.4.3-*.eb configs. Plus it can help to maintain local extra extensions easier.

Reply:
kind of include files.

Reply:
@boegel Regarding to the new R module, can you suggest me a more elegant solution than make this `R` easyconfig? `NCL` and `CDO`  must use `HDF5` 1.8, and `NCL` have `GDAL` dependency (with `HDF5` 1.8).

And what do you mean with 
```
will cause this module to be incompatible with others also depending on R (unless some clever swapping is done)
```
What are others also depending on `R`? Any example? (especially you want to use in combination with geo software?)

for the other extension, see https://github.com/easybuilders/easybuild-easyconfigs/issues/6355